plocket/al_story
# Fix #12, strings broken into arrays

Fix #12. To test use [extra_arrays_for_signature_fields.txt](https://github.com/plocket/al_story/files/6570604/extra_arrays_for_signature_fields.txt)

(To see the previous broken version, you can pull the gh-pages branch, which I really need to change to be the default branch)

Test:
1. Pull repo branch
2. Open index.html in browser
3. Copy contents of file into first text box
4. See that result is missing rows like this:
```
| signature_fields[0][0] | "u" | |
| signature_fields[0][1] | "s" | |
```

That would have spelled out `users[0].signature`, but it should no longer appear.
